drivers/client_access_control/app/controllers/client_access_control/history_controller.rb

```diff
@@ -24,6 +24,7 @@ def show
         [date.year, date.month]
       end.
         uniq
+      @month_index = params[:month_index].present? ? (params[:month_index].to_i || 0) : nil
```

Comment:
I _think_ this is equivalent to:
```suggestion
      @month_index = params[:month_index].to_i
```
since `nil.to_i` returns zero.